[qos_sai] portName referenced before assignment on t1-lag topo

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/sonic-net/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**

<!--
Briefly describe the problem you are having in a few paragraphs.
-->
PR [6393](https://github.com/sonic-net/sonic-mgmt/pull/6393) switched to using `portName` instead of `iface` for [t1 topos](https://github.com/sonic-net/sonic-mgmt/blob/7afbb6dbd7b0c21d2056c6be7e4279783f487131/tests/qos/qos_sai_base.py#L552).
Since portName is defined inside if loop, it results in `variable referenced before assignment` error if interfaces names do not contain '.'
**Steps to reproduce the issue:**
1. deploy t1-lag topo
2. run qos/test_qos_sai.py

**Describe the results you received:**
```
                if iface.startswith("Ethernet"):
                    if "." in iface:
                        portName, vlan_id = iface.split(".")
>                   portIndex = mgFacts["minigraph_ptf_indices"][portName]
E                   UnboundLocalError: local variable 'portName' referenced before assignment
```

**Describe the results you expected:**
Tests are passed.

**Additional information you deem important:**
<!--
software/ASIC/Hardware Flatform version and info
-->
    **Output of `show version`:**

```
SONiC Software Version: SONiC.master.172539-dirty-20221110.175553
Distribution: Debian 11.5
Kernel: 5.10.0-12-2-amd64
Build commit: 7c746e67d
Build date: Thu Nov 10 18:01:32 UTC 2022
Built by: AzDevOps@sonic-build-workers-002D7H

Platform: x86_64-accton_as9516_32d-r0
HwSKU: newport
ASIC: barefoot
ASIC Count: 1
```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

--- a/tests/qos/qos_sai_base.py
+++ b/tests/qos/qos_sai_base.py
@@ -551,6 +551,7 @@ class QosSaiBase(QosBase):
             for iface,addr in dut_asic.get_active_ip_interfaces(tbinfo).items():
                 vlan_id = None
                 if iface.startswith("Ethernet"):
+                    portName = iface
                     if "." in iface:
                         portName, vlan_id = iface.split(".")
                     portIndex = mgFacts["minigraph_ptf_indices"][portName]

